src/services/hospitalInfo/hospitalInfoService.ts

```diff
@@ -1,70 +1,73 @@
-import { airtableService } from '../../mapping/airtableService';
-import { HospitalInfo } from '../../models/hospitalInfo';
+import { airtableService } from "../../mapping/airtableService";
+import { HospitalInfo } from "../../models/hospitalInfo";
 
 import thumbnailData from "../../../test/thumbnailData.json";
 
 const extractUrls = (attachments: any) => {
-    return attachments && attachments.length > 0 ? attachments.map((att: any) => att.url) : [];
+  return attachments && attachments.length > 0
+    ? attachments.map((att: any) => att.url)
+    : [];
 };
 
 class MockHospitalInfoService {
-    async getHospitalInfo(): Promise<HospitalInfo[]> {
-        return thumbnailData.map(data => {
-            return {
-                id: data["ID"],
-                name: data["Hospital Name"],
-                status: data["Status"],
-                type: data["Type of Organization"],
-                description: data["Organization Notes / Description"],
-                year: data["Kids Served / Year"],
-                country: data["Country"],
-                state: data["State"],
-                zip: data["ZIP"],
-                city: data["City"],
-                address: data["Address"],
-                longitude: data["Longitude"],
-                latitude: data["Latitude"],
-                hospitalPicture1: extractUrls(data["Hospital Picture 1"][0]),
-                hospitalPicture2: extractUrls(data["Hospital Picture 1"][1]),
-                hospitalPicture3: extractUrls(data["Hospital Picture 1"][2]),
-            } as HospitalInfo
-        });
-    }
+  async getHospitalInfo(): Promise<HospitalInfo[]> {
+    return thumbnailData.map((data) => {
+      return {
+        id: data["ID"],
+        name: data["Hospital Name"],
+        status: data["Status"],
+        type: data["Type of Organization"],
+        description: data["Organization Notes / Description"],
+        year: data["Kids Served / Year"],
+        country: data["Country"],
+        state: data["State"],
+        zip: data["ZIP"],
+        city: data["City"],
+        address: data["Address"],
+        longitude: data["Longitude"],
+        latitude: data["Latitude"],
+        hospitalPicture1: extractUrls(data["Hospital Picture 1"]),
+        hospitalPicture2: extractUrls(data["Hospital Picture 1"]),
```

Comment:
Should the pic2 & pic3 refer to a different field?

Reply:
There is only Hospital Picture 1 in `test/thumbnailData`. `Airtable` has both Hospital Picture 2 and Hospital Picture 3. Would you like me to add Hospital Picture 2 and Hospital Picture 3 to `test/thumbnailData` as well?